drivers/gles3/storage/render_scene_buffers_gles3.cpp

```diff
@@ -33,17 +33,12 @@
 #include "render_scene_buffers_gles3.h"
 #include "texture_storage.h"
 
-#ifdef ANDROID_ENABLED
```

Comment:
Did you mean to remove the multiview support? or is it moved somewhere else?

Reply:
Its not used now that depth is initialized in the render target allocation code

Reply:
Yeah, it's not removed completely - just moved from the render buffers over to the render targets!

Reply:
Ah makes sense!